README.md

```diff
@@ -65,4 +65,10 @@ This repo uses [`just`](https://github.com/casey/just) as a command runner. Belo
 
 
 ### Contributing
-Each package's README contains in-depth information about the package's structure and suggestions on how add features specific to that package
\ No newline at end of file
+Each package's README contains in-depth information about the package's structure and suggestions on how add features specific to that package
+
+Before contributing, set up the pre-commit hook by running:
+
+```bash
+cp .githooks/pre-commit .git/hooks/pre-commit
+chmod +x .git/hooks/pre-commit
```

Comment:
very interesting... this markdown will render even tho you don't have the trailing ``` and I suspect it's because it's at the very end of the file? how strange

Reply:
rather than placing this in the `Contributing` section, how about in the [`Development` section](https://github.com/TBD54566975/web5-go/blob/b97ed95473ebb813f6d05b7976689b2ae8bf902c/README.md#development)? maybe underneath [`Prerequisites` section](https://github.com/TBD54566975/web5-go/blob/b97ed95473ebb813f6d05b7976689b2ae8bf902c/README.md#prerequisites) we have a section of the same header size called `Git Hooks`